src/Akeneo/UserManagement/Component/Model/User.php

```diff
@@ -152,6 +154,11 @@ class User implements UserInterface
     /** @var array $property bag for properties extension */
     private $properties = [];
 
+    private int $consecutiveAuthenticationFailureCounter=0;
+
+    /** @var DateTime|null */
+    private ?DateTime $authenticationFailureResetDate=null;
```

Comment:
```suggestion
    private ?\DateTime $authenticationFailureResetDate=null;
```

and the `use DateTime;` can be remove. (and from php_cd.php too)

Reply:
Done

Reply:
I tried you suggestion but I  still have to keep it in  php_cd.php